tests/cnf/core/network/security/tests/nftables-custom-firewall-test.go

```diff
@@ -106,9 +106,6 @@ var _ = Describe("nftables", Ordered, Label(tsparams.LabelNftablesTestCases), Co
 		By("Remove the static route to the external Pod network on each worker node")
 		addDeleteStaticRouteOnWorkerNodes(testPodList, routeMap, "del", hubIPv4Network)
 
-		By(fmt.Sprintf("Disables nftables on %s if active", cnfWorkerNodeList[0].Definition.Name))
```

Comment:
This is wrong. This will disable nftables AFTER reverting the custom nftables systemd file, which would then flush ALL rules. Consequently this will cause the issue you are seeing in CI.

Ensure that nftables is disabled before code in lines 118-120:

```
By(fmt.Sprintf("Remove machine-configuration %s", mcNftablesName))
err := mco.NewMCBuilder(APIClient, mcNftablesName).Delete()
Expect(err).ToNot(HaveOccurred(), "Failed to get the machineConfig")
```